proxysql/tests/test_proxysql.py

```diff
@@ -0,0 +1,163 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import mock
+import pytest
+from tests.conftest import PROXYSQL_VERSION
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.errors import ConfigurationError
+from datadog_checks.proxysql import ProxysqlCheck
+
+from .common import (
+    ALL_METRICS,
+    COMMANDS_COUNTERS_METRICS,
+    CONNECTION_POOL_METRICS,
+    GLOBAL_METRICS,
+    MEMORY_METRICS,
+    QUERY_RULES_TAGS_METRICS,
+    USER_TAGS_METRICS,
+)
+
+
+def get_check(instance):
```

Comment:
For when we have multiple test files, may want to do this now

Reply:
But as a fixture you need to set an extra param in each test function.
In that case there is a single test file, but even with multiple, we could import the helper method from somewhere and I think it's cleaner to have a single import than updating every test signature.

Reply:
by that logic, what's the point of fixtures for config?

Reply:
I think a fixture is useful if you execute code in it. You read a file maybe, or you do a deepcopy of an integration config. By that mean, it abstracts a lot of complexity in your test, you just use the result of that fixture as a parameter of your tests.

If the result of your fixture is a static function that you could define what's the point? Seems more complex (the test runs the code of the fixture, gets the result, put in in an argument of the test, then you call that argument (because the result of that fixture was a function itself)....) than simply defining the function statically.